README.md

```diff
@@ -144,3 +152,49 @@ This means that the CLI  needs to know the current version of the widget when us
 
 If you don't want to use the `--version` option on every update or deletion, the alternative is to use `--force`. When the `--force` option is present the CLI will automatically use the latest version of the widget. Be aware that using `--force` option might lead to accidental overwrites if multiple people are working on the same widget.
 
+### Programmatic usage
+
```

Comment:
When are you planning to extract the commands logic to a separate lib? I'm asking because I would rather not make this interface public as part of the `cli` if we are going to move it somewhere else in the near future.